lib/Backends/Interpreter/Interpreter.cpp

```diff
@@ -146,8 +146,7 @@ bool Interpreter::isOpSupported(const NodeInfo &NI) const {
 
   case Kinded::Kind::FullyConnectedNodeKind:
```

Comment:
I believe we should actually just remove this entire case for `Kinded::Kind::FullyConnectedNodeKind` -- it is not supported whether quantized or not. The only FullyConnected that is supported by the Interpreter is the RowwiseQuantized version, but it has its own case in here.

Reply:
updated it.